portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,6 +14,9 @@
 
 package com.google.sps.servlets;
 
+import java.util.ArrayList;
+import java.util.List;
+import com.google.gson.Gson;
```

Comment:
https://google.github.io/styleguide/javaguide.html
look at import order please (and I can also recommend reading through it once)